lib/axon/updates.ex

```diff
@@ -771,7 +771,7 @@ defmodule Axon.Updates do
   Adds random Gaussian noise to the input.
```

Comment:
The `trace` function in this file uses a decay-like algorithm with `:decay` defaulting to `0.9`. Should we use this default throughout?